DevCycle.SDK.Server.Local/ConfigManager/EnvironmentConfigManager.cs

```diff
@@ -186,11 +186,11 @@ private async Task FetchConfigAsyncWithTask()
             {
                 try
                 {
-                    var lastModified = res.Headers?.FirstOrDefault(e => e.Name?.ToLower() == "last-modified");
-                    var etag = res.Headers?.FirstOrDefault(e => e.Name?.ToLower() == "etag");
-                    if (!string.IsNullOrEmpty(configLastModified) && lastModified != null && !string.IsNullOrEmpty((string)lastModified.Value) )
+                    var lastModified = res.ContentHeaders?.FirstOrDefault(e => e.Name?.ToLower() == "last-modified")?.Value as string;
```

Comment:
what will the default be here if there's no "first"?

Reply:
null

Reply:
Will that be handled correctly during reading it or do we need a default of empty string?

Reply:
it's handled properly by the isnullorempty check below